docs/tools/swift.pod

```diff
@@ -1,30 +1,106 @@
 =pod
 
+=encoding utf8
+
 =head1 NAME
 
-swift - the amazingly new programming language
+swift - General purpose programming language by Apple Inc.
```

Comment:
"General purpose programming language" is super vague though...
Anyone else has other suggestions?